microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -22,145 +22,168 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # Edit files with Workspaces (Filesystem tab)
 
-This tutorial provides hands-on practice in setting up and using a Workspace.  After you add files to a Workspace, the changes that you make in your source code within DevTools are saved on your local computer, and are preserved after you refresh the webpage.
+This tutorial provides hands-on practice in setting up and using a workspace in DevTools. After you add files to a workspace changes that you make in your source code with DevTools are saved on your local computer. These changes are saved  after you refresh the webpage.
 
-Before beginning this tutorial, you should know how to do the following:
-* [Use html, CSS, and JavaScript to build a web page](https://developer.mozilla.org/docs/Learn/Getting_started_with_the_web)
-* [Use DevTools to make basic changes to CSS](../css/index.md)
-* [Run a local HTTP web server](https://developer.mozilla.org/docs/Learn/Common_questions/set_up_a_local_testing_server#Running_a_simple_local_HTTP_server)
+To refresh your knowledge of the technologies used, see the following articles:
 
+* Use HTML, CSS, and JavaScript to build a web page. See [Getting started with the web](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web)
 
-<!-- ====================================================================== -->
-## Overview
+* Use DevTools to make basic changes to CSS. See [Get started viewing and changing CSS](/microsoft-edge/devtools-guide-chromium/css/)
 
-Workspaces enable you to save a change that you make in Devtools to a local copy of the same file on your computer.  For this tutorial, you should have the following settings on your machine.
+* Run a local HTTP web server. See: 
 
-* You have the source code for your site on your desktop.
+  * Using Node.js (used for this tutorial): [Set up a localhost server](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/install#step-6-set-up-a-localhost-server)
+  
+  * Using Python: [Running a simple local HTTP server](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server#running_a_simple_local_http_server).
 
-* You are running a local web server from the source code directory, so that the site is accessible at `localhost:8080`.
 
-* You opened `localhost:8080` in Microsoft Edge, and you are using DevTools to change the CSS of the site.
+<!-- ====================================================================== -->
+## Introduction
 
-With Workspaces enabled, the CSS changes that you make within DevTools are saved to the source code on your desktop.
+A DevTools _workspace_ lets you save changes that you make to a local copy of the source code to the same file on your computer. For this tutorial, you need to have the following environment on your computer:
```

Comment:
```suggestion
A DevTools _workspace_ lets you save changes that you make to a local copy of the source code to the same file on your computer, so that changes are retained across refreshes of the page. Here's a typical scenario for using a workspace:
```